cristin-import/src/test/java/cucumber/ArtisticFeatures.java

```diff
@@ -110,4 +104,232 @@ public void theCristinResultLackTheDurationInBothMetadataFields() {
+    public void theOtherPerformanceHasAMusicalWPeWithTitleComposerThatIsAPremiere(String title, String composer) {
+        var musicalWorkPerformance = extractMusicPerformance();
+        var otherPerformance = (OtherPerformance) musicalWorkPerformance.getManifestations().get(0);
+        assertThat(otherPerformance.getMusicalWorks(), hasSize(1));
+        assertThat(otherPerformance.getMusicalWorks().get(0), instanceOf(MusicalWork.class));
+        var program = (MusicalWork) otherPerformance.getMusicalWorks().get(0);
+        assertThat(program.getComposer(), is(equalTo(composer)));
+        assertThat(program.getTitle(), is(equalTo(title)));
+    }
+
+    @And("the performance has a ISRC equal to {string}")
+    public void thePerformanceHasAnIsrcEqualTo(String isrc) {
+        scenarioContext.getCristinEntry().getCristinArtisticProduction().setIsrc(isrc);
+    }
+
+    @And("the performance has a medium equal to {string}")
+    public void thePerformanceHasAMediumEqualTo(String medium) {
+        scenarioContext.getCristinEntry().getCristinArtisticProduction().setMedium(medium);
+    }
+
+    @And("the performance has a publisher name equal to {string}")
+    public void thePerformanceHasAPublisherNameEqualTo(String publisherName) {
+        scenarioContext.getCristinEntry().getCristinArtisticProduction().setPublisherName(publisherName);
+    }
+
+    @Then("the Nva resource has a AudioVisualPublication")
+    public void theNvaResourceHasAAudioVisualPublication() {
+        var musicalWorkPerformance = extractMusicPerformance();
+        assertThat(musicalWorkPerformance.getManifestations(), hasItem(instanceOf(AudioVisualPublication.class)));
+    }
+
+    @And("the AudioVisualPublication has a mediaSubType equalTo {string},"
+        + " ISRC equalTo {string}, unconfirmedPublisher name equal to {string}")
+    public void theAudiovisualPublicationHasAMediaSubTypeEqualToExpectedValues(
+        String expectedMediaSubType,
+        String expectedIsrc,
+        String expectedUnconfirmedPublisher) throws InvalidIsrcException {
+        var musicalWorkPerformance = extractMusicPerformance();
+        var audioVisualManifestationOptional = musicalWorkPerformance
+            .getManifestations()
+            .stream()
+            .filter(manifestation -> manifestation instanceof AudioVisualPublication)
+            .findFirst();
+        assertThat(audioVisualManifestationOptional.isPresent(), is(equalTo(true)));
+        var audioVisualManifestation = (AudioVisualPublication) audioVisualManifestationOptional.get();
+        assertThat(audioVisualManifestation.getMediaType().getType().getValue(), is(equalTo(expectedMediaSubType)));
+        assertThat(audioVisualManifestation.getIsrc(), is(equalTo(new Isrc(expectedIsrc))));
+        var actualPublisher = (UnconfirmedPublisher) audioVisualManifestation.getPublisher();
+        assertThat(actualPublisher.getName(), is(equalTo(expectedUnconfirmedPublisher)));
+    }
```

Comment:
"Best practice" would be to split the various things into statements "And…has an X", but it isn't necessary to change this as I don't think these steps will be re-used

Reply:
In fact, the first step exists at the bottom of the file.

Reply:
Fixed here and for other places too.